src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/descriptionsHelper.hpp

```diff
@@ -138,8 +138,8 @@ class DescriptionsHelper final
         }
         else
         {
-            // Fallback to default ADP
-            vendorConfig = TGlobalData::instance().vendorMaps().at(ADP_DESCRIPTIONS_MAP_KEY).at(DEFAULT_ADP);
+            // Fallback to the default CNA
+            vendorConfig = TGlobalData::instance().vendorMaps().at(ADP_DESCRIPTIONS_MAP_KEY).at(DEFAULT_CNA);
```

Comment:
Is the comment a little bit confusing or am I misunderstanding what it means? 
The default CNA is CISA, but is not the fallback CNA.

Reply:
Removed